externs/es6.js

```diff
@@ -147,7 +147,6 @@ Math.sign = function(value) {};
 Math.cbrt = function(value) {};
 
 /**
- * @param {number} value1
  * @param {...number} var_args
```

Comment:
You need to delete the `value1` argument from the parameter list, too.

Reply:
@brad4d Nice catch! I've fixed.